README.md

```diff
@@ -30,66 +30,259 @@ From the GNU site:
+
+If Elasticsearch is disabled there will be no full-text search, however crawling and discovering new sites will still work.
+
+
+### ElasticSearch
+You will need to install Elasticsearch(probably not only the pip package), this is the link to download the last version of 5.x. : https://www.elastic.co/downloads/past-releases/elasticsearch-5-6-6 . You can have problems with versions (like I said in the warning section). If you want to be sure you are using the right version, you can do this command :
+
+    curl -XGET 'http://localhost:9200'
+
+To enable Elasticsearch
+
+    service elasticsearch start
+    ./elasticsearch_migrate.sh  #To perform the initial setup or if you want to reset Elasticsearch, but we need it at the beginning to start it. 
+
+After restart :
+
+    . venv/bin/activate
+    ./script/start.sh  #To start the instance of tor and privoxy
+
+### FLASK :
+    ./scripts/web.sh  #Launch flask to have a web interface
+
+### Cronjobs
+
+    #Harvest onions from various sources
+    1 18 * * * /home/freshonions-torscraper/scripts/harvest.sh
+
+    #Get ssh fingerprints for new sites
+    1 4,16 * * * /home/freshonions-torscraper/scripts/update_fingerprints.sh
+
+    #Mark sites as genuine / fake from the /r/darknetmarkets superlist
+    1 1 * * 1 /home/freshonions-torscraper/scripts/get_valid.sh
+
+    #Scrape pastebin for onions (needs paid account / IP whitelisting)
+    */5 * * * * /home/freshonions-torscraper/scripts/pastebin.sh
+
+    #Portscan new onions
+    1 13 * * * /home/freshonions-torscraper/scripts/portscan_up.sh
+
+    #Scrape stronghold paste
+    32 */2 * * * /home/freshonions-torscraper/scripts/stronghold_paste_rip.sh
+
+    #Detect clones
+    20 14 * * * /home/freshonions-torscraper/scripts/detect_clones.sh
+
+    #Keep a sql dump of data
+    1 */1 * * * mysqldump -u username -ppassword --database tor --result-file=/home/dump.sql
+    1 */8 * * * mysqldump -u username -ppassword --database tor --result-file=/home/dump_backup.sql
 
 
 ## Infrastructure
 
-Fresh Onions runs on two servers, a frontend host running the database and hidden service web site, and a backend host running the crawler. Probably most interesting to the reader is the setup for the backend. TOR as a client is COMPLETELY SINGLETHREADED. I know! It's 2017, and along with a complete lack of flying cars, TOR runs in a single thread. What this means is that if you try to run a crawler on a single TOR instance you will quickly find you are maxing out your CPU at 100%.
+Fresh Onions runs on two servers, a frontend host running the database and hidden service website, and a backend host running the crawler. Probably most interesting to the reader is the setup for the backend. T
+OR as a client is COMPLETELY SINGLETHREADED. I know! It's 2017, and along with a complete lack of flying cars, TOR runs in a single thread. What this means is that if you try to run a crawler on a single TOR instance you will quickly find you are maxing out your CPU at 100%.
```

Comment:
whitespace changes warning!

It looks like a newline was added in TOR (after the T) if this is the only change, it should be reverted. Whitespace changes contribute to nothing except git diff noise.